[GlobalISel][NFC]Delete the comments of XXLegalizerInfo

shining1984 opened the pull request:
Delete the LegalizerInfo comments of AArch64/AMD64/ARM/M68k/RISCV/x86, they are copied from register bank.

shining1984 committed:
[GlobalISel][NFC]Update the comments of XXLegalizerInfo

Update the LegalizerInfo comments of AArch64/AMD64/ARM/M68k/RISCV/x86,
according the BPFLegalizerInfo.

arsenm commented on llvm/lib/Target/ARM/ARMLegalizerInfo.h:
```diff
@@ -23,7 +23,8 @@ namespace llvm {
 class ARMSubtarget;
 
-/// This class provides the information for the target register banks.
+/// This class provides the information for the ARM target legalizer for
```
I'd just delete the comment from all the target implementations altogether

shining1984 replied:
I delete them.

shining1984 committed:
[GlobalISel][NFC]Delete the wrong comments of XXLegalizerInfo

Delete the LegalizerInfo comments of AArch64/AMD64/ARM/M68k/RISCV/x86,
they are copied from register banks.

RKSimon reviewed (approved):
LGTM
